scribejava-core/src/main/java/com/github/scribejava/core/model/OAuth2ErrorResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.github.scribejava.core.model;
+
+import java.net.URI;
+
+// https://tools.ietf.org/html/rfc6749#section-5.2
+public class OAuth2ErrorResponse {
```

Comment:
more info - easier to understand the problem.
And it will not add overhead in normal flow